gradle.properties

```diff
@@ -16,3 +16,5 @@ org.gradle.vfs.verbose=true
 
 # If you're experimenting with changes and don't want to update the verification file right away, please change the mode to "lenient" (not "off")
 org.gradle.dependency.verification=strict
+
+kotlin.stdlib.default.dependency=false
```

Comment:
❓ what's this about?

Reply:
See https://kotlinlang.org/docs/reference/using-gradle.html#dependency-on-the-standard-library